doc/main.html

```diff
@@ -179,7 +179,7 @@ <h4>pubPart</h4>
       </section>
 
       <section id="sec-pubVersion">
-        <h4>pubVersion</h4>
+        <h4><code>pubVersion</code></h4>
```

Comment:
```suggestion
        <h4><code>pubVersion</code></h4>
```

I am not in love with using `code` in headings since the heading merely a label for the actual contents.

Reply:
Good point, I'll remove and also update the `<code>` guidelines PR as well noting that.

Reply:
On second pass, as discussed offline, we will keep this as is for now. Example from this being used outside of SMPTE docs can be seen at https://html.spec.whatwg.org/multipage/urls-and-fetching.html#extracting-character-encodings-from-meta-elements